Plots/Trajectories/conda_environment.yml

```diff
@@ -0,0 +1,20 @@
+name: geocat-examples
```

Comment:
This file should be removed from the PR because there is only one `conda_environment.yml` which is located under the root directory. Thus, the geographiclib dependency should be added to that environment file instead. I have done  adding it by another PR #163 in favor of this PR because the dependency should be merged before this PR to get all the Sphinx-builds in the correct way. Since I have already added the dependency, it is good from now on that you either add the same dependency to conda environment as well or not.

Reply:
I think I should have this resolved and the other two comments about the "Trajectory" directory. Let me know if I missed anything!

Reply:
Yes, that's correct.